src/components/Blogs/index.jsx

```diff
@@ -3,24 +3,40 @@ import { Container, Row } from "react-bootstrap";
 import SearchBar from "../SearchBar";
 import BlogCard from "../BlogCard";
 import "./index.scss";
+import i18next from "i18next";
 
-const POSTS_PER_PAGE = 9;
+const POSTS_PER_PAGE = 18;
 const BLOG_API_URL = `https://public-api.wordpress.com/wp/v2/sites/safeplace102505649.wordpress.com/posts?per_page=${POSTS_PER_PAGE}`;
 
 export default function Blogs() {
   const [blogPosts, setBlogPosts] = useState([]);
   const [blogsForSearch, setBlogsForSearch] = useState([]);
-
-  const fetchBlogPosts = async () => {
-    const data = await fetch(BLOG_API_URL);
-    const fetchedBlogPosts = await data.json();
-    setBlogsForSearch(fetchedBlogPosts);
-    setBlogPosts(fetchedBlogPosts);
-  };
+  const [languageRef, setLanguageRef] = useState("10221");
 
   useEffect(() => {
+    const fetchBlogPosts = async () => {
+      const data = await fetch(BLOG_API_URL);
+      const fetchedBlogPosts = await data.json();
+      const filteredPosts = await fetchedBlogPosts.filter((post) => {
+        return post.tags.join() === languageRef;
+      });
+
+      setBlogsForSearch(fetchedBlogPosts);
+      setBlogPosts(fetchedBlogPosts);
+      setBlogPosts(filteredPosts);
+    };
+
     fetchBlogPosts();
-  }, []);
+  }, [languageRef]);
+
+  i18next.on("languageChanged", (lng) => {
+    let ref = i18next.translator.language;
+    if (ref === "en") {
+      setLanguageRef("10221");
```

Comment:
magic string 

please google / read technical manual

what does 10221 mean?
what does 38229 mean?

pretend a teammate just joined and has to read this code. would they understand?

Reply:
I was looking at this weird number :)

Reply:
fixed